plugins/Flanger/FlangerControls.cpp

```diff
@@ -88,3 +93,11 @@ void FlangerControls::changedPlaybackState()
 {
 	m_effect->restartLFO();
 }
+
+
+
+
+void FlangerControls::updatePhase()
+{
+	m_effect->m_lfo->setOffset(m_lfoPhaseModel.value() / 180 * D_PI);
```

Comment:
I did that a while back because putting it as FlangerEffect didn't work, I didn't know back then that the cause was FlangerEffect.h not being listed under MOCFILES in CMakeLists.txt.  Will fix.